tests/features/cli/CreateDatabaseCommand.test.ts

```diff
@@ -7,7 +7,8 @@ import { CreateDatabaseCommand } from '../../../packages/cli/src/commands/Create
 import { initORMSqlite } from '../../bootstrap';
 
 const closeSpy = jest.spyOn(MikroORM.prototype, 'close');
-const showHelpMock = jest.spyOn(CLIHelper, 'showHelp');
+const showHelpMock =
+  jest.spyOn(CLIHelper, 'showHelp');
```

Comment:
next time please dont make unrelated changes
```suggestion
const showHelpMock = jest.spyOn(CLIHelper, 'showHelp');
```